tasklib/tests.py

```diff
@@ -777,7 +780,7 @@ def test_deserializer_returning_empty_value_for_empty_string(self):
         # Test that any deserializer returns empty value when passed ''
         t = Task(self.tw)
         deserializers = [getattr(t, deserializer_name) for deserializer_name in
-                        filter(lambda x: x.startswith('deserialize_'), dir(t))]
+                         filter(lambda x: x.startswith('deserialize_'), dir(t))]
```

Comment:
List comprehensions can be "exploded" onto multiple lines to make them much clearer, like so:

```python
deserializers = [
    getattr(t, deserializer_name)
    for deserializer_name in filter(
        lambda x: x.startswith('deserialize_'),
        dir(t),
    )
]
```

In this case I also split the `filter` call onto multiple lines, but that's not strictly necessary as it'd fit on a single line.

Reply:
Fixed in 8f4fabf